src/lib/stores/index.ts

```diff
@@ -52,6 +52,13 @@ export const temporaryChatEnabled = writable(false);
 export const scrollPaginationEnabled = writable(false);
 export const currentChatPage = writable(1);
 
+export const feedbackDialog:Writable<FeedbackDialog> = writable({ open: false });
+
+type FeedbackDialog = {
+	open: boolean;
+	rating?: number
```

Comment:
suggestion: rating is now `number | undefined` but should he `numbern | null`

```suggestion
	rating: number | null
```

Reply:
and prettier says
```suggestion
	rating?: number;
};
````